manuscript/main.tex

```diff
@@ -506,21 +478,20 @@ \subsubsection*{Gut microbial taxonomic and functional profiles predict cognitiv
 genes. However, because the relationships between individual taxa are
 still largely unknown, we turned to Random Forest (RF) models, a family of 
 unsupervised non-parametric machine learning (ML) method that enables
-the identification of underlying patterns in large numbers of individual
-features (here, microbial species). Previous studies have reported
+the identification of underlying patterns in large numbers of individual features
+(here, microbial species).
```

Comment:
@vanjakle Can you explain when you want to change this to "taxa" instead of "species"? We're only testing species.

Reply:
Because you are only working with the sequence data, I thought it was more appropriate to call this a taxon. But, if you feel strongly about calling it a species and are confident in its classification, then go for it.